FILE packages/website/src/routes/index.tsx
RANGE {"path":"packages/website/src/routes/index.tsx","old_lines":[26,30],"new_lines":[26,30],"diff_block_lines":[5,11],"hunk_header":"@@ -26,5 +26,5 @@ function Home() {","substantive_changed_lines":2}
@@ -26,5 +26,5 @@ function Home() {
         style={{ backgroundImage: 'url(/hero-bg.jpg)' }}
       >
-        <div className="absolute inset-0 bg-background/80" />
+        <div className="absolute inset-0 bg-background/90" />
         <div className="absolute inset-x-0 bottom-0 h-32 bg-gradient-to-t from-black to-transparent" />
 
